components/Button/src/index.svelte

```diff
@@ -11,6 +11,8 @@
 	export let attrs = {}
 	export let type: IKunTypePro = 'primary'
 	export let disabled = false
+  export let size: 'normal' | 'small' | 'large' = 'normal'
```

Comment:
I don't really want to use normal small and large as the value of size, can you refer to this comment?
![image](https://github.com/ikun-svelte/ikun-ui/assets/32354856/7bbdd113-1dcf-4236-a5b1-ee56291b51aa)

Reply:
You only need to modify the name of the props and the name of the related class. Don’t refer to it for the value of the style, just design it reasonably.